CATALOG.md

```diff
@@ -35,3 +35,71 @@ nativefier https://www.udemy.com/
 ### Notes
 
 Most videos will work, but to be sure everything works you are better off using the `--widevine` version AND signing the app afterwards. See this post: [#1147 (comment)](https://github.com/nativefier/nativefier/issues/1147#issuecomment-828750362)
+
+
+## Spotify
+
+```
+nativefier https://open.spotify.com/
+--name "Spotify" 
+--widevine 
+-u "<useragent your browser uses>"
+--inject spotify.js
+--inject spotify.css
+--icon icon.
```

Comment:
Why this dot? Maybe you wanted to write

```suggestion
--icon icon.png
```

?

Reply:
should propably tell to set the according icon extension, mac uses a different format than windows that's why I left it out

Reply:
> should propably tell to set the according icon extension, mac uses a different format than windows that's why I left it out

@Knallli oooh true. Let's omit it, then. I'll remove it from the other ones. It's personal preference anyway.